tests/php/ORM/SQLSelectTest.php

```diff
@@ -435,29 +435,33 @@ public function testFiltersOnFK()
         );
     }
 
-    public function testInnerJoin()
+    public function testJoinSQL()
```

Comment:
This wasn't actually testing inner join - it was just testing the SQL output for both inner and left join. So I've just renamed it to match what it's actually testing, and added right join to it since it already had both existing join types in there.